src/sagemaker/sklearn/estimator.py

```diff
@@ -99,6 +100,9 @@ def __init__(
                 If ``framework_version`` or ``py_version`` are ``None``, then
                 ``image_uri`` is required. If also ``None``, then a ``ValueError``
                 will be raised.
+            image_uri_region (str): If ``image_uri` argument is None, the image uri
+                associated with this object will be in this region.
+                    Default: region associated with SageMaker session.
```

Comment:
@evakravi probably this is the indentation sphinx is complaining about. You can verify locally by running `tox -e sphinx,doc8 --parallel all`